suite-native/transactions/src/components/transactionsList/TransactionListItem.tsx

```diff
@@ -31,6 +39,11 @@ const transactionListItemStyle = prepareNativeStyle(utils => ({
 
 export const TransactionListItem = memo(({ transaction }: AccountTransactionListItemProps) => {
```

Comment:
Do you have a good reason to use `memo` here? I don't think that _transactions_ are rerendered very often. And this component is not even large...